gh-129948: Add `set()` to `multiprocessing.managers.SyncManager` (#129949)
python/cpython: Misc/NEWS.d/next/Library/2025-02-11-06-42-17.gh-issue-129948.ZcugY9.rst

> Added support for :func:`set` to :class:`multiprocessing.managers.SyncManager`.

Reviewer: ```suggestion
Add support for shared :class:`set` to :class:`multiprocessing.managers.SyncManager`
via :meth:`SyncManager.set() <multiprocessing.managers.SyncManager.set>`.
```

Reviewer: (I'm actually rewording my own suggestion :'))

Propose Feature: Support for :func:`set` in :class:`multiprocessing.managers.SyncManager`

I propose adding support for the :func:`set` function in the :class:`multiprocessing.managers.SyncManager`.

> :class:`multiprocessing.managers.SyncManager` have :func:`dict`, :func:`list` functions, but it does not have a :func:`set` function.

Reviewer: ```suggestion
Added support for :func:`set` to :class:`multiprocessing.managers.SyncManager`.
```

The rationale does not need to be explained in the blurb entry. It's more important to document the new feature instead (the issue would serve as the rationale).
